### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by CallMeSandra**
Created skeleton 
Added bootstrap, autoprefixer, changed database from sqlite to postgresql 
@agatasalska   , please check if everything's all right

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/db/*.sqlite3-journal
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore Byebug command history file.
+.byebug_history
```
Comment (agatasalska) [inline .gitignore:21]: I would add

# Ignore test-related files

/coverage.data
/coverage/

Comment (CallMeSandra) [inline .gitignore:21]: Done :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# Ignore test-related files
+/coverage.data
+/coverage/
```
Comment (agatasalska) [inline .gitignore:25]: ok imho

**Comment by CallMeSandra** [comment]
@mentors, @gotar

**Comment by gotar** [comment]
👍 you can merge

**Pull request merged**